smartmon.sh

```diff
@@ -68,6 +68,7 @@ workload_minutes
 SMARTMONATTRS
 )"
 smartmon_attrs="$(echo "${smartmon_attrs}" | xargs | tr ' ' '|')"
+smartctl=$(command -v smartctl)
```

Comment:
How is this different from simply omitting the absolute path from the `smartctl` command invocations, and letting the shell PATH find it?